src/app/pages/component-viewer/_component-viewer-theme.scss

```diff
@@ -14,6 +14,9 @@
     .mat-tab-label:focus {
       color: mat-color($foreground, text);
     }
+  }
 
+  .docs-api-h3 {
+    border-bottom: 1px solid rgba(mat-color($foreground, secondary-text), 0.15);
```

Comment:
How about?

```scss
1px solid mat-color($foreground, divider);
```

Reply:
Yeah, that's a little more in line with intention